packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts

```diff
@@ -674,8 +674,7 @@ export class CurrentActivityPrinter extends ActivityPrinterBase {
   public start() {
     // Need to prevent the waiter from printing 'stack not stable' every 5 seconds, it messes
     // with the output calculations.
-    this.oldLogLevel = logLevel;
-    setLogLevel(LogLevel.DEFAULT);
+    setLogLevel(LogLevel.INFO);
```

Comment:
okay for now but we will need to get rid of this eventually. `setLogLevel` should only every be set once as it affects the output. These messages just need to be logged with the correct level (or not at all).